yt/yt/client/api/operation_client.cpp

```diff
@@ -227,5 +227,56 @@ void TListOperationsAccessFilter::Register(TRegistrar registrar)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+TGetJobStderrResponse TGetJobStderrResponse::MakeJobStderr(const TSharedRef& data, const TGetJobStderrOptions& options)
+{
+    auto totalSize = std::ssize(data);
+    auto endOffset = totalSize;
+    auto offset = options.Offset.value_or(0);
+    auto limit = options.Limit.value_or(0);
+
+    if (!offset && !limit) {
```

Comment:
It is better to check that original ` options.Offset` and `options.Limit` are not set

Reply:
But if they set to 0 we should return same